react/components/EXPERIMENTAL_Conditions/Statement.js

```diff
@@ -68,13 +68,15 @@ class Statement extends React.Component {
       statements,
       statementIndex,
       labels,
+      onChangeObjectCallback,
     } = this.props
     const condition = statements[statementIndex]
     const atomProps = {
       statements: statements,
       options: options,
       isFullWidth: isFullWidth,
       statementIndex: statementIndex,
+      onChangeObjectCallback: onChangeObjectCallback,
```

Comment:
You could declare it without the `: onChangeObjectCallback` if the key has the same name as the variable.